translations/catroweb.en.yml

```diff
@@ -64,13 +64,31 @@ deleteAllNotificationsButtonTitle: "Delete all"
 notificationsClearError: "Unable to mark notifications as read!"
 notificationsDeleteError: "Unable to delete notifications!"
 pleaseWait: "Please wait..."
+followError: "Unable to follow the desired user!"
+unfollowError: "Unable to unfollow the desired user!"
+unfollowQuestion: "Are you sure you want to unfollow this user?"
+unfollowButton: "Unfollow"
```

Comment:
I think we should put this in the follower section below, too. Or in a new section.
Or maybe the multiline way of e.g. _programs.deleteConfirmation_ is a better?